bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -120,7 +126,8 @@ const defaultProps = {
 };
 
 const ALLOW_FULLSCREEN = Meteor.settings.public.app.allowFullscreen;
-const { isSafari } = browserInfo;
+const APP_CONFIG = Meteor.settings.public.app;
+const { isSafari, isMobileApp } = browserInfo;
```

Comment:
Please rename this variable to be called `isTabletApp` to match the new app name